charts/fluent-bit/values.yaml

```diff
@@ -141,6 +141,39 @@ resources: {}
 #     cpu: 100m
 #     memory: 128Mi
 
+## only available if kind is Deployment
+autoscaling:
+  enabled: false
+#  minReplicas: 1
+#  maxReplicas: 100
+#  targetCPUUtilizationPercentage: 80
+#  targetMemoryUtilizationPercentage: 80
+   ## see https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#autoscaling-on-multiple-metrics-and-custom-metrics
+  customRules: []
+#     - type: Pods
+#       pods:
+#         metric:
+#           name: packets-per-second
+#         target:
+#           type: AverageValue
+#           averageValue: 1k
+    ## see https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-configurable-scaling-behavior
+#   behavior:
+#      scaleDown:
+#        policies:
+#          - type: Pods
+#            value: 4
+#            periodSeconds: 60
+#          - type: Percent
+#            value: 10
+#            periodSeconds: 60
+
+## only available if kind is Deployment
+podDisruptionBudget:
+  enabled: false
+  annotations: {}
+#  maxUnavailable: "30%"
```

Comment:
Values used by name in a template should be defined, they can be empty with an example commented out below if required. If values have good defaults but are behind a switch they should be defined with that default.

Reply:
done

Reply:
@stevehipwell please check now